doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc

```diff
@@ -51,10 +51,10 @@ end
 ----
 +
 This business rule detects when a file named `invalid.pdf` is uploaded to the case file. It then removes the `invalid.pdf` document and creates a new instance of the `Prepare hardware spec` user task.
-. Click *Build*, then *Deploy* to deploy the IT Orders project.
+. Click *Build*, then *Deploy* to deploy the *IT_Orders* project.
```

Comment:
I think I've commented before that I would add another click.
Click Build then click Deploy to ...

Reply:
Done